worlds/mlss/Client.py

```diff
@@ -0,0 +1,292 @@
+                        await ctx.send_msgs(
+                            [
+                                {
+                                    "cmd": "Set",
+                                    "key": f"mlss_flag_{ctx.team}_{ctx.slot}",
+                                    "default": 0,
+                                    "want_reply": False,
+                                    "operations": [{"operation": "or", "value": 1 << (location - 0xDA0000)}],
+                                }
+                            ]
+                        )
+                        continue
+                    if location in roomException:
+                        if current_room not in roomException[location]:
+                            exception = True
+                        else:
+                            exception = False
+                    else:
+                        exception = True
+
+                    if location in eReward:
+                        if location not in self.eUsed:
+                            self.eUsed += [location]
+                            location = eReward[len(self.eUsed) - 1]
+                        else:
+                            continue
+                    if (location in ctx.server_locations) and exception:
+                        locs_to_send.add(location)
+
+            # Check for set location flags.
+            for byte_i, byte in enumerate(bytearray(flags)):
+                for j in range(8):
+                    if j in self.checked_flags[byte_i]:
+                        continue
+                    and_value = 1 << j
+                    if byte & and_value != 0:
+                        flag_id = byte_i * 8 + (j + 1)
+                        room, item = find_key(roomCount, flag_id)
+                        pointer_arr = await bizhawk.read(
+                            ctx.bizhawk_ctx, [(ROOM_ARRAY_POINTER + ((room - 1) * 4), 4, "ROM")]
+                        )
+                        pointer = struct.unpack("<I", pointer_arr[0])[0]
+                        pointer = pointer & 0xFFFFFF
+                        offset = await bizhawk.read(ctx.bizhawk_ctx, [(pointer, 1, "ROM")])
+                        offset = offset[0][0]
+                        if offset != 0:
+                            offset = 2
+                        pointer += (item * 8) + 1 + offset
+                        for key, value in beanstones.items():
+                            if pointer == value:
+                                pointer = key
+                                break
+                        if pointer in ctx.server_locations:
+                            self.checked_flags[byte_i] += [j]
+                            locs_to_send.add(pointer)
+
+            if not ctx.finished_game and cackletta != 0 and current_room == 0x1C7:
+                await ctx.send_msgs([{"cmd": "StatusUpdate", "status": ClientStatus.CLIENT_GOAL}])
+
+            await ctx.send_msgs(
```

Comment:
Same here, sending a `Set` 8 times per second to update the room could be done only when the current room has changed.